src/views/AgmLocationChg.vue

```diff
@@ -382,7 +430,7 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
 
   get previousYear () : number {
     const today = new Date()
-    return new Date(today.getFullYear() - 1, today.getMonth() + 1, today.getDate()).getFullYear()
+    return new Date(today.getFullYear() - 2, today.getMonth() + 1, today.getDate()).getFullYear()
```

Comment:
![system - 2](https://github.com/bcgov/business-filings-ui/assets/122301442/059bff40-cd29-4e97-b246-29766af9ed70)

Reply:
I'm sure there's an easier way to do this...

Reply:
Nicely done 👍